misk/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -60,6 +90,22 @@ internal class RealTransacter(
     }
   }
 
+  private fun isRetryable(e: Exception): Boolean {
+    // TODO(jmuia): check the causal chain.
```

Comment:
yeah, and a test case that confirms that we retry the right ones!

Reply:
I think I'll do this in a follow up